handlers/validator.go

```diff
@@ -1729,32 +1765,60 @@ func ValidatorHistory(w http.ResponseWriter, r *http.Request) {
 			})
```

Comment:
1 line above this the `tableData` has 4 entries but shouldn't it be 3 like below?
It probably makes no functional difference.

Reply:
fixed in new commit, ptal